taxi/tests/test_forms.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm
+
+
+class FormsTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.username = "test_user"
+        cls.password = "Test1234!"
+        cls.first_name = "James"
+        cls.last_name = "Bond"
+
+    def create_form(self, license_number: str):
+        form_data = {
+            "username": self.username,
+            "password1": self.password,
+            "password2": self.password,
+            "first_name": self.first_name,
+            "last_name": self.last_name,
+            "license_number": license_number,
+        }
+        return DriverCreationForm(form_data)
```

Comment:
The create_form method is creating a new instance of the form with every call. This potentially reduces the performance of your tests. You could consider using the setUp method to create a single instance of the form and reuse it in all tests.